### ISSUE TITLE
Use a custom LLM

### ISSUE THREAD

**Issue by webdevatn** [livekit/agents#700]
I have an LLM ready and deployed on a server. Can I use my apis to get the response instead of openAI.llm()?

**Comment by lenage** [comment]
please see [minimal_assistant.py](https://github.com/livekit/agents/pull/714/files#diff-59487b56c139dc19ee686bc300f6765ba983a182fb4387a708cc939642b85b6a) as example

**Comment by ChrisFeldmeier** [comment]
Yes you can! I had the same problem and solved it thourgh developing a own additional LLM module. and I also passed through the language. It's not the beste way but it worked for me.
![SCR-20240909-aj](https://github.com/user-attachments/assets/1d2ab9cc-b069-4722-9a51-54694084b46b)

+1 for livekit devs to implement this
